charts/forms-flow-all-in-one/values.yaml

```diff
@@ -0,0 +1,26 @@
+---
+global:
+  Domain:  #<DEFINE_ME>
+
+forms-flow-admin:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-ai:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-analytics:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-api:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-bpm:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-data-analysis:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-documents-api:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-forms:
+  Domain: "{{.Values.global.Domain}}"
+forms-flow-idm:
+  keycloak:
+    ingress:
+      hostname: "forms-flow-idm-{{.Release.Namespace}}.{{tpl (.Values.global.Domain) .}}"
+forms-flow-web:
```

Comment:
We will need to bring all values in forms-flow-ai/values.yml here too ?

Reply:
Each chart works with sane default values; only ```.Values.Domain```  needs to be defiend during the install.
This all-in-one chart implicitly installs all the subcharts with their defauly values.yaml,  which will provide an environment similar to the install script.

The benefits to an all-in-one chart are:
- Stable version releases.  
- Subchart inheritance.
- Application and microservice decoupling.  
- Optional microservices (keycloak, postgres, etc)